examples/gallery/symbols/points_cities.py

```diff
@@ -0,0 +1,50 @@
+"""
+GeoPandas: Plotting points with Point or MultiPoint geometry
+============================================================
+
+The :meth:`pygmt.Figure.plot` method allows us to plot geographical data such as points
+with Point or MultiPoint geometry types stored in a :class:`geopandas.GeoDataFrame`
+object. Use :func:`geopandas.read_file` to load data from any supported OGR format such
+as a shapefile (.shp), GeoJSON (.geojson), geopackage (.gpkg), etc. Then, pass the
+:class:`geopandas.GeoDataFrame` object as an argument to the ``data`` parameter of
+:meth:`pygmt.Figure.plot`, and style the points using the ``fill`` and ``pen``
+parameters. Additional pass suitable columns of the :class:`geopandas.GeoDataFrame` to
+the ``x``,  ``y``, and ``text`` parameters of the :meth:`pygmt.Figure.text` method to
+label specific features.
+"""
+
+# %%
+import geopandas as gpd
+import pygmt
+
+# Read a sample dataset provided by Natural Earth. The dataset contains cities stored
+# as Point geometry type. In this example we focus on Europe.
+provider = "https://naciscdn.org/naturalearth"
+cities = gpd.read_file(f"{provider}/50m/cultural/ne_50m_populated_places_simple.zip")
+cities = cities[cities["name"] != "Vatican City"].copy()  # No overlap with label Rome
```

Comment:
I prefer to removing this line, since this is just a gallery example showing the usage and it doesn't have to be perfect.
```suggestion
```